.travis.yml

```diff
@@ -7,6 +7,15 @@ os: osx
 # on Traivs.
 env:
   - TEST_DIR=com.google.code.gson-gson
+  - TEST_DIR=org.joda-joda-convert
+  - TEST_DIR=joda-time-joda-time
+
+matrix:
+  allow_failures:
+    # Blocked on https://github.com/JodaOrg/joda-convert/issues/7
```

Comment:
Namely, this is a snapshot only:
https://github.com/JodaOrg/joda-convert/blob/master/pom.xml#L12

When release time comes, depending on the API changes the maintainer might need to call if 2.0, 1.7.1, etc.